tests/helpers.ts

```diff
@@ -0,0 +1,6 @@
+import faker from 'faker';
+import { IEntity } from '../src/entity/models/entity';
+
+export const createFakeEntity = (): IEntity => {
+  return { externalId: faker.random.uuid(), osmId: faker.random.number() };
+};
```

Comment:
I think this should be nested under `tests/unit/entity/helpers.ts`, wdyt?

Reply:
Because this helper is used by both integration and unit tests it should be outside, but I moved it into a folder for more clarity.